rfcs/0009_Unified_Static_Memory_Planning.md

```diff
@@ -348,12 +350,27 @@ tvmc compile my_model.tflite --executor=aot --output-format=mlf --target=c
 
 This should be a IRModule (TIR) → IRModule (TIR) pass.
 
-Inputs : 
-* AoT TIR PrimFunc ( the control function describing the call graph to operators)
-* All Operator Functions
-* the maximum size for each pool We could use "pinned_memory" (see below) to tag buffers with suggested priority order determined by the scheduler.
+Inputs :
+* IRModule containing 
+    * AoT TIR PrimFunc (the control function describing the call graph to operators)
+    * All Operator Functions
+    * Each tir.allocate in the IRModule annotated with candidate pools ([Using the annotation field of tir.allocate](https://github.com/apache/tvm-rfcs/blob/c447cbfbd5abceaa7623a0f90cc492784e6f0c0b/rfcs/0023-adding-annotation-field-to-tir.allocate.md))
+
+
+```
```

Comment:
er, this is kind of just here without much explanation. it makes sense in light of the comments threads here..but can you add something more explicit to indicate that this is an input? even if we are not quite sure how this is going to land and just defer to that other RFC, it's helpful to state that here.